Formula/g/glibc.rb

```diff
@@ -148,9 +203,37 @@ def install
         "--without-selinux",
         "--with-binutils=#{bootstrap_dir}/bin",
         "--with-headers=#{Formula["linux-headers@5.15"].include}",
+        "--with-bugurl=#{tap.issues_url}",
+        "--with-pkgversion=Homebrew glibc (#{pkg_version})",
       ]
-      system "../configure", *args
+
+      cflags = "-O2 #{ENV["HOMEBREW_OPTFLAGS"]}"
+      cflags += " -mbranch-protection=standard" if Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
+
+      system "../configure", *args, "CFLAGS=#{cflags}"
       system "make", "all"
+
+      if build.bottle?
+        # Some tests need a libgcc_s.so to be present.
+        # However this must be one built with glibc - unlike the bootstrap one built without glibc.
+        ln_s Utils.safe_popen_read("/usr/bin/cc", "-print-file-name=libgcc_s.so.1").chomp, Pathname.pwd
+
+        # We do break the ABI in one unavoidable way.
+        # This is because `_nl_default_dirname` varies based on the install prefix.
+        sysv_dir = if Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
+          "aarch64"
+        elsif Hardware::CPU.intel? && Hardware::CPU.is_64_bit?
+          "x86_64/64"
+        end
+        if sysv_dir
+          inreplace "../sysdeps/unix/sysv/linux/#{sysv_dir}/libc.abilist",
+                    /(_nl_default_dirname.*?)0x12/,
+                    "\\10x40"
```

Comment:
Maybe something like `("#{Formula["glibc"].share}/locale".length + 1).to_s(16)` given the length will change based on version/revision.

Reply:
fluid ABI sounds worse but seems like it's tricky to separate build and install locale directories

Reply:
I've got a fix, just deciding whether it makes sense to be `opt_share` or `HOMEBREW_PREFIX/share`

Reply:
Forgot about the keg-only status.

As a precaution I'll probably make it `opt_share`. It will probably mean things don't get picked up from packages, but at the same time system glibc likely behaves exactly the same way so it would at least be consistent.

Plus nobody ever reported a bug about it.